hybrasyl/CreatureStatus.cs

```diff
@@ -94,6 +100,13 @@ public interface ICreatureStatus
         void OnEnd();
 
     }
+   
+    public class StatusInfo
+    {
+        public string Name;
+        public string CastableName;
+        public double Remaining;
+    }
```

Comment:
Is this a simple data structure? If so, Castable name should be removed, and proper getter/setter should be specified on the properties.

Reply:
Will fix.